src/AugmentedMatrix.jl

```diff
@@ -0,0 +1,62 @@
+using LinearAlgebra
+
+#Used in one line in function below
+function invert_or_zero(x :: Real)
+    if x == 0
+        return 0
+    else
+        return 1/x
+    end
+end
+
+#Code to compute augmented rate matrix
+function augmentedRateMatrix(list_of_rate_matrices :: Array{B}, list_of_time_steps :: Array{T}) where {T <: Real, B <: Matrix{T}}
```

Comment:
In general it's better to use a 3 dimensional `Array` instead of an `Array` of other `Arrays` so it gets allocated contiguously.